### PR TITLE
Increment version in library.properties

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

**Comment by pablogs9** [comment]
@Acuadros95 can you take a look at this? How are we versioning foxy and galactic branch? Can we use the naming `vx.y.z-[ros_version]` in the Arduino `library.properties` file?

**Comment by Acuadros95** [comment]
@pablogs9 Yes, we can use `version=2.0.3-foxy`

File: library.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name=micro_ros_arduino
-version=0.0.4
+version=2.0.3
```
Comment (Acuadros95) [inline library.properties:2]:
```suggestion
version=2.0.3-galactic
```

**Comment by amilcarlucas** [comment]
Changes done

**Review by pablogs9 — APPROVED** [review]

**Comment by amilcarlucas** [comment]
done, 2 new PRs

**Pull request merged**